packages/ndla-ui/src/model/WordClass.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const WordClass = {
+  Adjective: 'adjective',
+  Adverb: 'adverb',
+  ProperNoun: 'proper noun',
+  Auxiliary: 'auxiliary',
+  Complement: 'complement',
+  Conjunction: 'conjunction',
+  Coverb: 'coverb',
+  Determiner: 'determiner',
+  Interjection: 'interjection',
+  Quantifier: 'quantifier',
+  Marker: 'marker',
+  ModalVerb: 'modal verb',
+  MeasureWord: 'measure word',
+  Noun: 'noun',
+  'Noun-zh': 'noun',
```

Comment:
Det er vel verdien som er nøkkelen her?
```suggestion
  'Noun-zh': 'noun-zh',
```